core/src/main/kotlin/com/exactpro/th2/read/db/bootstrap/Main.kt

```diff
@@ -179,18 +220,30 @@ private fun createReader(
     return reader
 }
 
-private fun TableRow.toMessage(dataSourceId: DataSourceId): RawMessage.Builder {
-    return RawMessage.newBuilder()
+private fun TableRow.toProtoMessage(dataSourceId: DataSourceId): ProtoRawMessage.Builder {
+    return ProtoRawMessage.newBuilder()
         .setBody(UnsafeByteOperations.unsafeWrap(toCsvBody()))
         .apply {
             sessionAlias = dataSourceId.id
-            direction = Direction.FIRST
+            direction = ProtoDirection.FIRST
             associatedMessageType?.also {
                 metadataBuilder.putProperties("th2.csv.override_message_type", it)
             }
         }
 }
 
+private fun TableRow.toTransportMessage(dataSourceId: DataSourceId): RawMessage.Builder {
+    val builder = RawMessage.builder()
+        .setBody(Unpooled.wrappedBuffer(toCsvBody()))
+        .setId(MessageId(dataSourceId.id, Direction.INCOMING, 0, Instant.EPOCH))
```

Comment:
I think we should use `idBuilder` here (because we cannot set sequence and timestamp here)